pkg/server/apis/v1alpha1/descriptors/tenant.go

```diff
@@ -88,4 +88,27 @@ var tenant = []definition.Descriptor{
 			},
 		},
 	},
+	{
+		Path:        "/tenants/{tenant}/precheck",
```

Comment:
"/tenants/{tenant}/executioncontexts" to list all execution contexts for the given tenant.

Reply:
done